### DISCUSSION TITLE
Turborepo 2 not using new terminal UI

### DISCUSSION THREAD

**Discussion by tylerlaberge** [vercel/turborepo#8763] [Help] [answered]
### Summary

According to [this blog post](https://turbo.build/blog/turbo-2-0) a new terminal UI was released/stabilized with turborepo 2

However, I'm not seeing how to enable this new UI. I'm running turborepo 2.0.6 and I'm still seeing the same UI that I've always seen, looks like this 

![image](https://github.com/user-attachments/assets/43ed3721-114c-4e41-be0c-0baa9e9abb03)

It's my understanding that I should be seeing a more interactive task UI, not sure what I'm doing wrong

Using

turborepo: 2.0.6
pnpm: 9.5.0

This is my turbo.json

```{
  "$schema": "https://turbo.build/schema.json",
  "tasks": {
    "build": {
      "dependsOn": ["^build"],
      "outputs": ["dist/**"]
    },
    "lint": {},
    "dev": {
      "cache": false,
      "persistent": true
    }
  }
}`

**Comment by tylerlaberge** [comment, marked as answer]
Found the answer immediately after posting this, there is a config setting to enable the new UI https://turbo.build/repo/docs/reference/configuration#ui

Perhaps this should be mentioned on the blog post? Or made more visible in the documentation? I think the new UI is very cool but it took me a while to find this

**Reply by anthonyshew** [reply]
The setting was disabled as the default in 2.0.9 due to bugs and will be re-enabled in a coming version soon. Using the configuration, as you've mentioned, would be the correct way to start using it again.
